modules/instrument_manager/php/instrument_manager.class.inc

```diff
@@ -87,29 +87,24 @@ class Instrument_Manager extends \NDB_Menu_Filter
                 array('testname' => $_FILES['install_file']['name'])
             );
             if ($exists <= 0) {
-                $db_config = $config->getSetting('database');
-                exec("php generate_tables_sql_and_testNames.php < $new_file");
-
-                $instrument = \NDB_BVL_Instrument::factory(
-                    $instname,
-                    '',
-                    ''
+                $error_message = exec(
+                    'php generate_tables_sql_and_testNames.php < '
+                    . escapeshellarg($new_file)
                 );
-                if (!isset($db_config['quatUser'], $db_config['quatPassword'])) {
+                if (!empty($error_message)
+                    && strpos($error_message, "Instrument") !== false
```

Comment:
What is the purpose of this check? If the error doesn't include the word Instrument shouldn't it still be thrown?

Reply:
The check was added so future php errors wouldn't suppressed by the LorisException. 
Example: someone is modifying the generate_tables_sql_and_testNames.php and writes some code resulting in a php error. The error message would get lumped into the LorisException without showing the php console warning.